shared/src/business/entities/cases/Case.test.js

```diff
@@ -2039,6 +2163,27 @@ describe('Case entity', () => {
   });
 
   describe('removePrivatePractitioner', () => {
+    it('does not remove a practitioner from associated case privatePractitioners array', () => {
```

Comment:
This should probably say something like "does not remove a practitioner **if not found in the** associated case privatePractioners array"

Reply:
Good catch, updated!